src/autocomplete.js

```diff
@@ -607,15 +629,8 @@ class Autocomplete extends Component {
     }
   }
 
-  componentDidUpdate(prevProps, prevState) {
-    // TODO: what do we need to do for this when the
-    // autocomplete is a controlled component?
-    if (
-      prevState.highlightedIndex !== this.state.highlightedIndex ||
-      this.state.selectedValue !== prevState.selectedValue
-    ) {
-      this.updateStatus()
-    }
+  componentDidUpdate() {
+    this.updateStatus()
```

Comment:
Turns out we want to update the status of the state anytime there's an update to the state 😅